lawnchair/src/app/lawnchair/LawnchairLauncher.kt

```diff
@@ -407,8 +407,10 @@ class LawnchairLauncher : QuickstepLauncher(), LifecycleOwner,
      */
     private fun forceReloadIcons() {
         val iconPack = prefs.iconPackPackage.get()
-        prefs.iconPackPackage.set("")
-        prefs.iconPackPackage.set(iconPack)
+        if (iconPack != "") {
+            prefs.iconPackPackage.set("")
+            prefs.iconPackPackage.set(iconPack)
+        }
```

Comment:
Revert or use below?

```suggestion
        if (iconPack != "") {
            prefs.iconPackPackage.set("")
        }
        prefs.iconPackPackage.set(iconPack)
```

Reply:
Why? we don't have to set it to the original value if we never change it.

Reply:
Sorry, missread it.